cmd/uplink/cmd/root.go

```diff
@@ -86,6 +87,11 @@ func (cliCfg *UplinkFlags) NewUplink(ctx context.Context) (*libuplink.Uplink, er
 		SkipPeerCAWhitelist: !cliCfg.TLS.UsePeerCAWhitelist,
 		PeerCAWhitelistPath: cliCfg.TLS.PeerCAWhitelistPath,
 	}
+
+	// We use longer timeouts for cmd/uplink to account for slower connections.
+	libuplinkCfg.Volatile.DialTimeout = time.Minute * 2
```

Comment:
won't this affect all connections, not only slow?

Reply:
Shouldn't this be:
```
	libuplinkCfg.Volatile.DialTimeout = cliCfg.Client.DialTimeout
	libuplinkCfg.Volatile.RequestTimeout = cliCfg.Client.RequestTimeout
```

Reply:
Yeah but normal/fast connections complete before the timeout whether it's `2m` or the previous `20s`.

Reply:
Ok, in the past timeout was set to low as possible (20sec) because in some flows one hanged storage node could increase whole cp/rm time. I'm not sure if its still a case.

Reply:
Yeah I think in the future we want to read all the settings from the config file that way and ship different config defaults for audit/repairs than for CLI/mobile clients but we don't have that in place right now. This is a temporary fix to override the defaults for `cmd/uplink` to allow the CLI to have longer timeouts. If there's another way to do this I'm open to suggestions!

Reply:
I'm not 100% but satellite transport will have default 20 sec one way or another https://github.com/storj/storj/blob/f56cb1c6123ce752458bc7e29e12326f49c6971c/satellite/peer.go#L249

Reply:
Good call! Nobody calls `transport.NewClientWithTimeouts()` so I can make the CLI client do that call specifically and leave the Satellite using the default path that inherits `20s` timeouts. I've pushed changes with your suggestions!